docs/api/gavertilfrivillige.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Gaver til frivillige API
+slug: /api/gavertilfrivillige
+folder: api
+sidebar: mydoc_sidebar
+datatable: true
+tags: [ API, gaver, frivillige, frivillighet, tro, livssyn ]
+keywords: [ grunnlagsdata ]
+last_updated: Dec 12, 2024
+hide_table_of_contents: true
+---
+
+<Summary>Tjeneste for innrapportering av tredjepartsopplysninger for gaver til visse frivillige organisasjoner og enkelte tros og livssynssamfunn.</Summary>
+
+<Tabs underline={true}>
+<TabItem headerText="Om tjenesten" itemKey="itemKey-1" default>
+
+For generell informasjon om tjenestene se egne sider om:
+
+* [Bruk av tjenestene](../om/bruk.md)
+* [Sikkerhetsmekansimer](../om/sikkerhet.md)
+* [Systemtilgang](../om/systemtilgang.md)
+* [Feilhåndtering](../om/feil.md)
+* [Versjonering](../om/versjoner.md)
+* [Teknisk spesifikasjon](../om/tekniskspesifikasjon.md)
+
+## Scope
+
+Følgende scope skal benyttes ved autentisering i Maskinporten: `skatteetaten:innrapporteringgaverfrivilligeorganisasjoner`
+
+## Delegering
+
+Tilgang til dette API-et kan delegeres i Altinn, f.eks. dersom leverandør benyttes for den tekniske oppkoblingen.
+
+## Systemtilgang
+
+Bruk av API-et krever systemtilgang, som er ny funksjonalitet i Maskinporten levert av Digdir. 
+Informasjon vedr. dette finnes [her](../om/systemtilgang.md). 
+
+## Teknisk spesifikasjon
+
+URL-er til API-et, beskrivelsen av parameterne, endepunkter og respons ligger i Open API spesifikasjonen på
+[SwaggerHub](https://app.swaggerhub.com/apis/skatteetaten/gavertilfrivillige-api/0.0.1) TODO: oppdater
+
+Oppbygning av URL-er og åpninger i en evt. brannmur er beskrevet her [URL-er](../om/url.md)
+
+API-et for innrapportering av gaver til frivillige har to endepunkter
```

Comment:
Vi har kanskje bare beskrivelse for 2 endepunkter i bolig og? Det er vel strengt tatt 3?

Reply:
Nei det er bare to igjen her. Metadata endepunktet er fjernet og bsu har ikke med det midlertidige dialog endepunktet som krevdes i pilot.